config/default.yml

```diff
@@ -34,4 +34,7 @@ public:
 
   handleInternalErrors: true
 
+  # 1,048,576 bytes = 1MB
+  maxResponseSizeBytes: 1048576
```

Comment:
I had a look to see if I can find a library that allows us to declare a nice string here like `maxResponseSize: "512kB"` or `maxResponseSize: "100MB"` and parse it into a number of bytes, but I couldn't find one. They all seem to go the other way (i.e: from bytes to human-readable notation). Any ideas?

Reply:
Found one!

https://github.com/visionmedia/bytes.js#bytesparsestringnumber-value-numbernull